log4j-core/src/main/java/org/apache/logging/log4j/core/pattern/package-info.java

```diff
@@ -18,7 +18,7 @@
  * Provides classes implementing format specifiers in conversion patterns.
  */
 @Export
-@Version("2.21.1")
+@Version("2.22.0")
```

Comment:
_Nit_: When a minor version bump is required in the package, we usually bump it to version number of the next minor Log4j version (`2.24.0`).

This convention helps us remembering, in which Log4j version the package was modified last.

Reply:
I see.

@vy should I bump it now again or better leave it at this point?